--- old/usr/src/man/man3secdb/getexecattr.3secdb	2014-05-26 17:39:16.743527670 +0100
+++ new/usr/src/man/man3secdb/getexecattr.3secdb	2014-05-26 17:39:16.750099532 +0100
@@ -131,7 +131,7 @@
 Once a list of entries is returned by \fBgetexecuser()\fR or
 \fBgetexecprof()\fR, the convenience function \fBmatch_execattr()\fR can be
 used to identify an individual entry. It returns a pointer to the individual
-element with the same profile name ( \fIprofname\fR), type name ( \fItype\fR),
+element with the same profile name (\fIprofname\fR), type name (\fItype\fR),
 and \fIid\fR. Function parameters set to \fINULL\fR are not used as part of the
 matching criteria. In the event that multiple entries meet the matching
 criteria, only a pointer to the first entry is returned. The
@@ -149,7 +149,7 @@
 .LP
 The \fBgetexecattr()\fR, \fBgetexecuser()\fR, and \fBgetexecprof()\fR functions
 all allocate memory for the pointers they return. This memory should be
-deallocated with the \fBfree_execattr()\fR call. The \fBmatch_execattr()\fR(
+deallocated with the \fBfree_execattr()\fR call. The \fBmatch_execattr()\fR
 function does not allocate any memory. Therefore, pointers returned by this
 function should not be deallocated.
 .sp
