--- old/usr/src/man/man3socket/getaddrinfo.3socket	Thu May 22 21:03:26 2014
+++ new/usr/src/man/man3socket/getaddrinfo.3socket	Thu May 22 21:03:26 2014
@@ -299,7 +299,7 @@
 .LP
 For \fBgetaddrinfo()\fR, if the query is successful, a pointer to a linked list
 of one or more \fBaddrinfo\fR structures is returned by the fourth argument and
-the function returns \fB0\fR. The order of the addresses returned i nthe fourth
+the function returns \fB0\fR. The order of the addresses returned in the fourth
 argument is discussed in the ADDRESS ORDERING section. If the query fails, a
 non-zero error code will be returned. For \fBgetnameinfo()\fR, if successful,
 the strings hostname and service are copied into \fIhost\fR and \fIserv\fR,
